app/assets/stylesheets/active_admin/print.scss

```diff
@@ -164,62 +134,21 @@ table {
 
 // Show
 .panel {
-  border-bottom: thick solid #ccc;
   margin-bottom: 3em;
   padding-bottom: 2em;
   page-break-inside: avoid;
-
-  &:last-child {
-    border-bottom: none;
-  }
 }
 
-.comments {
-  form {
-    display: none;
-  }
-
-  .active_admin_comment {
-    border-top: thin solid black;
-    padding-top: 1em;
-
-    .active_admin_comment_meta {
-      h4 {
-        font-size: 1em;
-        font-weight: bold;
-        float: left;
-        margin-right: .5em;
-        margin-bottom: 0;
-      }
-
-      span {
-        font-size: .9em;
-        font-style: italic;
-        vertical-align: top;
-      }
-    }
-
-    .active_admin_comment_body {
-      clear: both;
-      margin-bottom: 1em;
-    }
-  }
+.active_admin_comment {
+  padding: 1em 0;
 }
 
+.active_admin_comment_meta {
```

Comment:
Selector `active_admin_comment_meta` should be written in lowercase with hyphens